test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -45,6 +45,7 @@ bool createRandomTest()
 		RandomCodeOptions options;
 		TestOutputHelper testOutputHelper;
 		std::string test = test::RandomCode::fillRandomTest(suite, c_testExampleStateTest, options);
+		std::cout << test << "\n";
```

Comment:
Why is an output being added in a PR "do not print ..."?  I'm confused.

Reply:
do not print in unit test. but print in --createRandomTest  option. I just moved the print statement.